test/RachisTests/DatabaseCluster/AtomicClusterReadWriteTests.cs

```diff
@@ -628,4 +599,102 @@ private static IDisposable LocalGetDocumentStores(string[] urls, string database
             return disposable;
         }
     }
+
+    public  class AtomicClusterReadWriteTests : AtomicClusterReadWriteTestsBase
+    {
+        public AtomicClusterReadWriteTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+
+        [Fact]
+        public async Task ClusterWideTransaction_WhenStore_ShouldCreateCompareExchange()
+        {
+            await base.ClusterWideTransaction_WhenStore_ShouldCreateCompareExchange();
+        }
+
+        [Fact]
+        public async Task ClusterWideTransaction_WhenDisableAndStore_ShouldNotCreateCompareExchange()
+        {
+            await base.ClusterWideTransaction_WhenDisableAndStore_ShouldNotCreateCompareExchange();
+        }
+
+        [Fact]
+        public async Task ClusterWideTransaction_WhenLoadAndUpdateInParallel_ShouldSucceedOnlyInTheFirst()
+        {
+            await base.ClusterWideTransaction_WhenLoadAndUpdateInParallel_ShouldSucceedOnlyInTheFirst();
+        }
+
+        [Fact]
+        public async Task ClusterWideTransaction_WhenLoadAndDeleteWhileUpdated_ShouldFailDeletion()
+        {
+            await base.ClusterWideTransaction_WhenLoadAndDeleteWhileUpdated_ShouldFailDeletion();
+        }
+
+        [Fact(Skip = "Should complete shard implementation")]
```

Comment:
please unskip those when `https://github.com/ravendb/ravendb/pull/13671` merged

Reply:
This got merged